datahub-web-react/src/app/entityV2/shared/tabs/Incident/AcrylComponents/IncidentDrawerHeader.tsx

```diff
@@ -50,7 +56,7 @@ export const IncidentDrawerHeader = ({
                         <Tooltip2 title="Edit Incident">
```

Comment:
here you should say:

<Tooltip2 title={canEditIncidents ? "Edit Incident" : noPermissionMessage} />

Reply:
Also, should make the button type / state = Disabled. So the user cannot click it